TELCODOCS-1705-416 MetalLB: Drop the legacy AddressPool (#74330)
openshift/openshift-docs: release_notes/ocp-4-16-release-notes.adoc

With this release, IP addresses and ranges are handled more efficiently in NSGs for {product-title} clusters hosted on Azure. As a result, the maximum limit of CIDRs for all Ingress Controllers in Azure clusters, using the `allowedSourceRanges` field, increases from approximately 1000 to 4000 CIDRs.

[id="ocp-4-16-metallb-addresspool-removed"]
==== MetalLB AddressPool custom resource definition (CRD) removed

> The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated for a while now. However, in this release, the CRD is completely removed. The sole supported method of configuring MetalLB address pools is by using the `IPAddressPools` CRD. 

Reviewer: Nit - do you think this entry could benefit with a slightly more formal version of "for a while now"? It could just be me but it reads just a bit more conversational than normal. If you don't have a specific version to reference here, maybe you could try something like this:
```suggestion
The MetalLB `AddressPool` custom resource definition (CRD) has been deprecated for several version. However, in this release, the CRD is completely removed. The sole supported method of configuring MetalLB address pools is by using the `IPAddressPools` CRD. 
```
